exercises/practice/protein-translation/protein_translation.f90

```diff
@@ -0,0 +1,14 @@
+module protein_translation
+  implicit none
+
+contains
+
+  function proteins(rna) result(names)
+    character(len=*), intent(in) :: rna
+    character(len=13), allocatable :: names(:)
+
+    names = [character(len=13) :: rna] ! Replace this line in your implementation
```

Comment:
This line doesn't make much sense. I would probably replace it with an empty array now, but I'm not sure if it's worth the effort at this point.
```fortran
names = [character(len=13) :: ] ! Replace this line in your implementation
```

Reply:
You can make that a suggestion in the review view, and show the change, it may be committed from that suggestion.

Reply:
Thanks! I'm fairly new to this, so I really appreciate little tips like this!

I updated my comment accordingly. But I also realised in the meantime that this would provoke a compiler warning, which isn't great. I leave it up to the reviewers to decide which variant is better.